eva-lib/src/main/java/uk/ac/ebi/eva/lib/json/ConsequenceTypeMixin.java

```diff
@@ -16,12 +16,14 @@
 
 package uk.ac.ebi.eva.lib.json;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
 import java.util.Set;
 
+@JsonIgnoreProperties({"sift", "polyphen"})
```

Comment:
Is it possible to check the document structure in the web service test to ensure that "sift" and "polyphen" are hidden and the structure of "proteinSubstitutionScores" is correct if present?